types/node/index.d.ts

```diff
@@ -5781,45 +5781,46 @@ declare module "crypto" {
         setAAD(buffer: Buffer): this;
     }
     /** @deprecated since v10.0.0 use createCipheriv() */
-    export function createDecipher(algorithm: string, password: any): Decipher;
-    export function createDecipheriv(algorithm: string, key: any, iv: any): Decipher;
+    export function createDecipher(algorithm: string, password: string | ArrayBufferView, options?: stream.TransformOptions): Decipher;
+    export function createDecipheriv(algorithm: string, key: string | ArrayBufferView, iv: string | Bufferalike, options?: stream.TransformOptions): Decipher;
     export interface Decipher extends NodeJS.ReadWriteStream {
-        update(data: Buffer | DataView): Buffer;
+        update(data: ArrayBufferView): Buffer;
         update(data: string, input_encoding: HexBase64BinaryEncoding): Buffer;
-        update(data: Buffer | DataView, input_encoding: any, output_encoding: Utf8AsciiBinaryEncoding): string;
+        update(data: ArrayBufferView, input_encoding: any, output_encoding: Utf8AsciiBinaryEncoding): string;
         update(data: string, input_encoding: HexBase64BinaryEncoding, output_encoding: Utf8AsciiBinaryEncoding): string;
         final(): Buffer;
         final(output_encoding: string): string;
         setAutoPadding(auto_padding?: boolean): this;
-        setAuthTag(tag: Buffer): this;
-        setAAD(buffer: Buffer): this;
+        setAuthTag(tag: ArrayBufferView): this;
+        setAAD(buffer: ArrayBufferView): this;
     }
-    export function createSign(algorithm: string): Signer;
+    export function createSign(algorithm: string, options?: stream.WritableOptions): Signer;
     export interface Signer extends NodeJS.WritableStream {
-        update(data: string | Buffer | DataView): Signer;
-        update(data: string | Buffer | DataView, input_encoding: Utf8AsciiLatin1Encoding): Signer;
+        update(data: string | ArrayBufferView): Signer;
+        update(data: string | ArrayBufferView, input_encoding: Utf8AsciiLatin1Encoding): Signer;
         sign(private_key: string | { key: string; passphrase: string }): Buffer;
         sign(private_key: string | { key: string; passphrase: string }, output_format: HexBase64Latin1Encoding): string;
     }
-    export function createVerify(algorith: string): Verify;
+    export function createVerify(algorith: string, options?: stream.WritableOptions): Verify;
     export interface Verify extends NodeJS.WritableStream {
-        update(data: string | Buffer | DataView): Verify;
-        update(data: string | Buffer | DataView, input_encoding: Utf8AsciiLatin1Encoding): Verify;
-        verify(object: string | Object, signature: Buffer | DataView): boolean;
+        update(data: string | ArrayBufferView): Verify;
+        update(data: string | ArrayBufferView, input_encoding: Utf8AsciiLatin1Encoding): Verify;
+        verify(object: string | Object, signature: Bufferalike): boolean;
```

Comment:
Why is here `Bufferalike` and not `ArrayBufferView` used?

Reply:
@Flarna I missed a replacement value. Originally I added Bufferalike as just all of the ArrayBuffers that I knew of and I found out about ArrayBufferView after that …